src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -403,18 +404,20 @@ public void deleteControlledResource(
    * across the four types of controlled resources.
```

Comment:
nit: this method doesn't build a list across resource types. it returns the list for a single type, and the aggregation happens elsewhere